component-samples/demo/aio/context.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Context>
   <Realm className="org.apache.catalina.realm.MemoryRealm" pathname="tomcat-users.xml"/>
+  <Valve className="org.apache.catalina.valves.ErrorReportValve" showReport="false" showServerInfo="false"/>
```

Comment:
Masking required for for both debug and release traces ?

Reply:
Yes